src/lineup/ARankingView.ts

```diff
@@ -77,13 +77,13 @@ export abstract class ARankingView extends AView {
    * clears and rebuilds this lineup instance from scratch
    * @returns {Promise<any[]>} promise when done
    */
-  protected rebuild = debounceAsync(() => this.rebuildImpl(), 100);
+  protected rebuild: () => Promise<void> = debounceAsync(() => this.rebuildImpl(), 100);
```

Comment:
The depicted typings do not match with the `@returns` above, what is the correct return value then?

Reply:
Good point. I fixed it.